All-in-one setup/createform.ps1

```diff
@@ -656,17 +669,37 @@ try {
             }else{
                 throw "Could not match right '$($permission)' to FullAccess, SendAs or SendOnBehalf"
             }
-            HID-Write-Status -Event Success -Message "Added permission $($permission) to mailbox $($identity) for $User."
-            HID-Write-Summary -Event Success -Message "Added permission $($permission) to mailbox $($identity) for $User."
+            Write-Information -Message "Added permission $($permission) to mailbox $($identity) for $User."
+
+            $Log = @{
+                Action            = "MailboxPermissions" # optional. ENUM (undefined = default) 
+                System            = "ExchangeOnline" # optional (free format text) 
+                Message           = "Added permission $($permission) to mailbox $($identity) for $User." # required (free format text) 
+                IsError           = $false # optional. Elastic reporting purposes only. (default = $false. $true = Executed action returned an error) 
+                TargetDisplayName = $User # optional (free format text) 
```

Comment:
DisplayName = userid and username = mailbox. This is incorrect

Reply:
In my opinion this is logged correctly. mailbox = $identity and user = $user

Reply:
$identity = $datasource.selectedmailbox.id , Identity links to the ID of the mailbox, so you would expect the displayname of that same target (identity) to be the displayname of that mailbox ie $datasource.selectedmailbox.Name or something like that?